### ISSUE TITLE
Make machinery usable for complex strings

### ISSUE THREAD

**Issue by bugzilla-to-github** [mozilla/pontoon#2125]
*This issue was created automatically by a [script](https://github.com/mathjazz/bugzilla2github/).*
## [Bug 1453341](https://bugzilla.mozilla.org/show_bug.cgi?id=1453341)
Bug Reporter: @mathjazz
CC:   fios@foramnagaidhlig.net, @gion-andri, @gaby2300, @jordis, @zbraniecki
Depends on:   [Bug 1576120](https://bugzilla.mozilla.org/show_bug.cgi?id=1576120)

We need to figure out how to use machinery with complex strings.

Complex strings have more than one value, e.g.:
- multiple attributes
- value + attribute(s)
- plural and other variants etc.

Machinery on the other hand only takes one value when making a request to its services.

Possible solutions:

1. Make requests to Machinery for each input field separately (on focus).

2. Investigate the possibility of only showing one input field at the same time in rich editor (similar as we do with Gettext plurals).

**Comment by mathjazz** [comment]
Superseded by https://github.com/mozilla/pontoon/issues/2886 and https://github.com/mozilla/pontoon/issues/3679.
